scripts/benchmarks_ci.py

```diff
@@ -377,13 +380,14 @@ def __run_benchview_scripts(args: list, verbose: bool) -> None:
     if 'Configuration' not in benchview_config:
         benchview_config['Configuration'] = args.configuration
 
-    # Generate existing configs. This may be a good time to unify them?
+    # Generate existing configs.
+    # TODO: Unify configs across all repos?
     submission_architecture = args.architecture
     if args.category.casefold() == 'CoreClr'.casefold():
-        benchview_config['JitName'] = 'ryujit'  # This is currently fixed.
+        benchview_config['JitName'] = 'ryujit'  # FIXME: Remove this.
```

Comment:
why aren't we tracking things like this with issues (which would be more discoverable)?

Reply:
We should, but I would like to talk about it first.